terragrunt/aws/cloud_asset_inventory/efs.tf

```diff
@@ -14,3 +14,11 @@ resource "aws_efs_mount_target" "neo4j" {
   subnet_id       = element(var.vpc_private_subnet_ids, count.index)
   security_groups = [aws_security_group.cartography.id]
 }
+
+resource "aws_efs_backup_policy" "policy" {
```

Comment:
Could you rename this resource so it references `neo4j` in it's name.  Something like:
```
resource "aws_efs_backup_policy" "neo4j"
```
Because we're using Terragrunt, we wouldn't have a name clash here with the other `aws_efs_backup_policy` resource named `policy` that you added, but if both these resources ended up in the same module, Terraform would fail with a name clash error.